base/applications/rapps/lang/es-ES.rc

```diff
@@ -212,11 +212,16 @@ BEGIN
     IDS_INTEG_CHECK_FAIL "El paquete no ha pasado la comprobación de integridad, puede haber sido alterado o estar corrupto. No se recomienda ejecutarlo."
     IDS_INTERRUPTED_DOWNLOAD "Se ha interrumpido la descarga por un error de red. Compruebe la conexión a Internet."
     IDS_UNABLE_TO_WRITE "No se ha podido escribir en el disco duro, es posible que no quede espacio libre."
+    IDS_SELECT_ALL "Select/Deselect All"
```

Comment:
delete line 215-218 since it already there
`-    IDS_SELECT_ALL "Select/Deselect All"`
`-    IDS_INSTALL_SELECTED "Install Selected"`
`-    IDS_SELECTEDFORINST "Selected for installation"`
`-    IDS_MISMATCH_CERT_INFO "The certificate used is unknown:\nSubject: %s\nIssuer: %s\nDo you want to continue anyway?"`